### PR TITLE
bin/build-docker-de: rename build-ccde and use build-cored-release

### PR DESCRIPTION
**Pull request by boymanjor**
bin/build-ccde has been renamed to build-docker-de. It
now uses bin/build-cored-release to build go binaries for
the developer edition docker image. Static version tags for
the binaries and the docker image are declared. Non-release
images can be built by passing image and cored version tags
as env vars. The docker image has also been updated to pin
the alpine version and remove the use of schema.sql.

### CODE REVIEW

**Comment by boymanjor** [comment]
@jeffomatic PTAL

File: bin/build-docker-de [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  rm -f $CHAIN/docker/de/cored
+  rm -f $CHAIN/docker/de/corectl
+  rm -f $CHAIN/docker/de/build-cored-release
+}
+trap cleanup EXIT
+
+imageVersion=${IMAGE_VERSION:-1.0.0}
+coredVersion=${CORED_VERSION:-cmd.cored-1.0.1}
+
+cp bin/build-cored-release $CHAIN/docker/de
```
Comment (jeffomatic) [inline bin/build-docker-de]: Is there a reason to copy this script into the target directory? It should be runnable from anywhere.

Comment (boymanjor) [inline bin/build-docker-de]: Oh, good catch. This is remnants from my earlier work compiling the binaries inside the image.

**Comment by boymanjor** [comment]
@jeffomatic updated.

**Pull request merged**